.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-1.9.3
```

Comment:
Octokit v4 requires Ruby >= 2.0
https://github.com/octokit/octokit.rb#upgrading-guide

We should upgrade the Ruby we develop with to a modern supported version like 2.1.x or 2.2.x